test/cypress/integration/templates/template_management_spec.js

```diff
@@ -12,55 +12,24 @@ describe('Template Management', () => {
         pages: ['catalog', 'blueprints']
     };
     const users = [defaultUser];
-    const builtInTemplates = [
-        {
-            id: 'main-default',
-            pages: ['adminDash', 'catalog', 'blueprints', 'deploy', 'sites', 'systemResources', 'logs'],
-            roles: 'default',
-            tenants: ['all']
-        },
-        {
-            id: 'main-sys_admin',
-            pages: ['adminDash', 'catalog', 'blueprints', 'deploy', 'sites', 'tmm', 'ha', 'systemResources', 'logs'],
-            roles: 'sys_admin',
-            tenants: ['all']
-        }
-    ];
-    const builtInPages = [
-        { id: 'adminDash', name: 'Dashboard' },
-        { id: 'blueprint', name: 'Blueprint' },
-        { id: 'blueprints-community', name: 'Local Blueprints' },
-        { id: 'blueprints', name: 'Local Blueprints' },
-        { id: 'catalog', name: 'Cloudify Catalog' },
-        { id: 'deploy', name: 'Deployments' },
-        { id: 'deployment', name: 'Deployment' },
-        { id: 'drilldownDeployments', name: 'Subdeployments' },
-        { id: 'execution', name: 'Logs' },
-        { id: 'ha', name: 'Admin Operations' },
-        { id: 'logs', name: 'Logs' },
-        { id: 'plugins', name: 'Plugins' },
-        { id: 'sites', name: 'Site Management' },
-        { id: 'systemResources', name: 'System Resources' },
-        { id: 'tmm', name: 'Tenant Management' }
-    ];
-
-    const verifyTemplateRow = (index, id, pages, roles, tenants) => {
-        const templateRow = `.blue.segment > .gridTable > :nth-child(2) > .very > tbody > :nth-child(${index})`;
-        cy.get(`${templateRow} > :nth-child(1)`).should('have.text', id);
-        cy.get(`${templateRow} > :nth-child(2)`).should('have.text', roles);
-
-        cy.get(`${templateRow} > :nth-child(1)`).click();
-        cy.get('.horizontal > :nth-child(1)').within(() => {
-            for (let i = 0; i < pages.length; i += 1) {
-                cy.get(`.divided > :nth-child(${i + 1})`).should('have.text', pages[i]);
-            }
-        });
-        cy.get('.horizontal > :nth-child(3)').within(() => {
-            for (let i = 0; i < tenants.length; i += 1) {
-                cy.get(`.divided > :nth-child(${i + 1})`).should('have.text', tenants[i]);
-            }
+
+    const verifyTemplateRow = (id, pages, roles, tenants) => {
+        cy.contains('tr', id).as('templateRow');
+        cy.get('@templateRow').within(() => {
+            roles.forEach(role => cy.get(`td:nth-of-type(2)`).should('contain.text', role));
```

Comment:
We usually use `contains(role)` instead of `should('contain.text', role)`. Is there a reason we use this new way here :thinking: ?

Reply:
If not, we could (optional) probably shorten it to:

```suggestion
            roles.forEach(role => cy.contains(`td:nth-of-type(2)`, role);
```

(I did not check it myself)

Reply:
Hmmm... Could be, but I'm not convinced it's better (see my comment: https://github.com/cloudify-cosmo/cloudify-stage/pull/1333#discussion_r621972646 ).